src/pages/publication/DescriptionPanel.tsx

```diff
@@ -161,20 +163,20 @@ const DescriptionPanel: FC<DescriptionPanelProps> = ({ goToNextTab, savePublicat
           <StyledFieldHeader>{t('description.project_association')}</StyledFieldHeader>
 
           <StyledFieldWrapper>
-            <Field name={DescriptionFieldNames.PROJECTS}>
-              {/* TODO: Use <FieldArray /> */}
-              {({ field: { value, name } }: any) => (
+            <FieldArray name={DescriptionFieldNames.PROJECTS}>
+              {({ name, insert, remove }) => (
                 <>
                   <ProjectSearch
-                    setValueFunction={newValue => setFieldValue(name, newValue ?? emptyProject)}
-                    value={value.title}
+                    setValueFunction={newValue => insert(0, newValue)}
                     dataTestId="search_project"
                     placeholder={t('description.search_for_project')}
                   />
-                  {value.title && <p>{value.title}</p>}
+                  {getObjectValueByFieldName(values, name).map((project: Project, i: number) => (
+                    <ProjectRow key={project.id} project={project} onClickRemove={() => remove(i)} />
```

Comment:
nødvendig å sette key her? kan du ikke heller flytte det til ProjectRow der du uansett har tilgang til alt i project?

Reply:
må dessverre ligge på hovedelementet i en map(). Hvis jeg flytter til `ProjectRow` får man `Warning: Each child in a list should have a unique "key" prop.` i console :(

Reply:
ahh oki